code/modules/clothing/under/miscellaneous.dm

```diff
@@ -594,6 +594,13 @@
 	item_color = "villain"
 	can_adjust = 0
 
+/obj/item/clothing/under/vine
+	name = "dress of vines"

```

Comment:
based on the sprite I would go with 'botanical halter top'

desc = "Any dates a good date after the contact high"